scripts/delayed.js

```diff
@@ -62,7 +62,10 @@ if (window.hlx.rum.isSelected) {
   document.head.appendChild(script);
 }
 
-if (document.querySelector('.article-header') && !document.querySelector('[data-origin]')) {
+// no-interlinks is a special "tag" to skip interlink via content
+if (document.querySelector('.article-header')
+  && document.querySelector('[data-origin]')
```

Comment:
I think this should still be `!document.querySelector('[data-origin]')`. if there are already interlinks on the page, it shouldn't run again.